### ISSUE TITLE
plus(Iterable<FileCollection>) warning

### ISSUE THREAD

**Issue by petarov** [johndevs/gradle-vaadin-plugin#209]
Hi,

Not really a show stopper, but these warning messages started appearing with Gradle 2.8 when using the vaadin-plugin. I'm not sure how far Gradle 3.0 is from being released, but I was just wondering where do they come from exactly.

```
Using Gradle Vaadin Plugin 0.10.2
The plus(Iterable<FileCollection>) method and using the '+' operator in conjunction with an Iterable<FileCollection> object have been deprecated and are scheduled to be removed in 3.0.  Please use the plus(FileCollection) method or the '+' operator with a FileCollection object instead.
```

With --debug on:

```
09:48:38.437 [WARN] [org.gradle.internal.featurelifecycle.LoggingDeprecatedFeatureHandler] The plus(Iterable<FileCollection>) method and using the '+' operator in conjunction with an Iterable<FileCollection> object have been deprecated and are scheduled to be removed in 3.0.  Please use the plus(FileCollection) method or the '+' operator with a FileCollection object instead.
```

**Comment by johndevs** [comment]
The warnings comes from the fix we made for Gradle 2.0 in #116. They are a cause of the following change made in Gradle 2.8 in https://docs.gradle.org/2.8/release-notes#adding-a-one-item-list-to-a-filecollection

**Comment by petarov** [comment]
Thank you for the details!
